core/include/usBundleContext.h

```diff
@@ -91,10 +92,61 @@ class US_Core_EXPORT BundleContext
 
 public:
 
-  BundleContext(const BundleContext&) = delete;
-  BundleContext& operator=(const BundleContext&) = delete;
+  BundleContext();
 
-  ~BundleContext();
```

Comment:
I'd simply like to confirm that removing the user-defined copy ctor/assignment operator and destructor was intentional to enable implicitly generated copy/move ctor and assignment operators.
Was this the intent?